src/Data/Auth/Providers/JwtAuthProvider.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext, useEffect, useState } from "react";
+import { useGetCurrentUser } from "@/Data/Managers/V2/GetCurrentUser";
+import { AuthContext } from "../AuthContext";
+
+/**
+ * Component that implements a authentication provider when jwt authentication is enabled.
```

Comment:
Might be good to add some more explanation to it? 
Something in the genre of what Bart described?
if the method is jwt you assume that a user is logged in and you show the username by calling the current_user api endpoint

Reply:
elaborated a bit more on it, I think now it's clear